_tests/test_createJson.py

```diff
@@ -0,0 +1,141 @@
+#!/usr/bin/env python
+
+# Copyright (C) 2022 Noelia Ruiz Martínez, NV Access Limited
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+import unittest
+import os
+import shutil
+import json
+from _validate import (
+	createJson,
+	addonManifest
+)
+
+TOP_DIR = os.path.abspath(os.path.dirname(__file__))
+SOURCE_DIR = os.path.dirname(TOP_DIR)
+INPUT_DATA_PATH = os.path.join(SOURCE_DIR, '_tests', 'testData')
+VALID_JSON = os.path.join(
+	INPUT_DATA_PATH,
+	"addons",
+	"fake",
+	"13.0.0.json"
+)  # json file available in testData/fake
+ADDON_PACKAGE = os.path.join(INPUT_DATA_PATH, 'fake.nvda-addon')
+MANIFEST_FILE = os.path.join(INPUT_DATA_PATH, 'manifest.ini')
+ADDON_CHANNEL = "testChannel"
+ADDON_PUBLISHER = "testPublisher"
+ADDON_SOURCE_URL = "https://example.com/"
+
+OUTPUT_DATA_PATH = os.path.join(SOURCE_DIR, '_tests', 'testOutput')
+
+
+def getAddonManifest():
+	with open(MANIFEST_FILE) as f:
+		manifest = addonManifest.AddonManifest(f)
+	return manifest
+
+
+class IntegrationTestCreateJson(unittest.TestCase):
+	""" Integration tests.
+	- The JSON file is created (written to the filesystem).
+	- The output is then loaded and checked for correctness.
+	"""
+	def setUp(self):
+		self.outputDir = os.path.join(OUTPUT_DATA_PATH, "createJsonOutput")
+		self.maxDiff = None
```

Comment:
```suggestion
```

Reply:
This is used by `unittest.Testcase` if not set you only get a small sample of the diff for the dictionaries.

Reply:
This might be worth adding a comment to explain